src/pages/payment-methods/index.tsx

```diff
@@ -13,6 +12,10 @@ import { localize, WithIntl, Localize } from 'components/localization'
 import { DerivStore } from 'store'
 import device from 'themes/device'
 
+type StyledTableInterface = {
```

Comment:
As per our convention we are naming `types` ending with `Type` 

https://wikijs.deriv.cloud/e/en/Frontend/deriv-static/06-implementation-guide/ts-convention

Reply:
okay